assessment.sql

```diff
@@ -5,24 +5,49 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees(
+  id SERIAL PRIMARY KEY
+  surname CHARACTER VARYING
+  given_name CHARACTER VARYING
+  gender CHARACTER VARYING
+  height INTEGER
+  weight INTEGER
+  dob DATE
+  eye_color CHARACTER VARYING
+  hair_color CHARACTER VARYING
+);
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
 
--- your code here
+INSERT INTO licensees(id,surname,given_name,gender,height,weight,dob,eye_color,hair_color)
+VALUES('Bishop','Caren','f',63,132,'1943-09-26','Brown','Black');
 
 -- Question 3. Load the data from `data/licensees.csv` into the `licensees` table.
 
--- your code here
+\copy licensees(id,surname,given_name,gender,height,weight,dob,eye_color,hair_color) FROM 'data/licensees.csv' WITH (HEADER true,FORMAT csv);
```

Comment:
That CSV file doesn't have a column with ID numbers. Listing `id` in the list of columns to set breaks this statement. Again, it is otherwise good. 3/4